backend/open_webui/retrieval/loaders/external_document.py

```
@@ -0,0 +1,58 @@
+import requests
+import logging
+from typing import Iterator, List, Union
```

Ruff (0.12.2)

`typing.Iterator` imported but unused

Remove unused import

(F401)